extensions/css-language-features/README.md

```diff
@@ -4,18 +4,20 @@ This extension offers CSS/SCSS/Less support in VS Code.
 
 ## Development
 
-- `yarn`
-- `yarn` at `/server`
-- Run the `Launch Extension` debug target in the Debug View
-- Run `Reload Window` command in the launched instance to reload the extension
+- Clone the `vscode` repository
+- Open the `extensions/css-language-features/` directory in VS Code.
+- Run `yarn` in `css-language-features/`.
+- Run `yarn` in `css-language-features/server/`.
+- Run the `Launch Extension` debug target in the Debug View.
```

Comment:
This should maybe clarify what _should_ happen? E.g. "Run the `Launch Extension` debug target in the Debug View to open a new VS Code window with your code changes."

Reply:
OK, that's a good point.